.jshintrc

```diff
@@ -0,0 +1,4 @@
+{
+  "esnext": true,
+  "node": true
+}
```

Comment:
What's the purpose of running jshint as well as eslint? Currently the CI only runs eslint (using rules from `.eslintrc.json`).